app/javascript/shared/constants/messages.js

```diff
@@ -58,39 +58,41 @@ export const ALLOWED_FILE_TYPES_FOR_TWILIO_WHATSAPP =
 
 export const CSAT_RATINGS = [
   {
-    key: 'disappointed',
```

Comment:
@iamsivin I didn't understand this change. The score 1 is for disappointed faced.